package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "react-simple-scroll",
+  "version": "0.0.3",
```

Comment:
Given how long we've been using this in production, I'd support calling this release `1.0.0` rather than treating it as pre-1.0